client/src/components/App/App.module.scss

```diff
@@ -0,0 +1,10 @@
+@import '../../styles/variables.scss';
+@import '../../styles/mixins.scss';
+
+.wrapper {
+  background-color: $bgColor;
+  width: 100vw;
+  height: 100vh;
+  .container {
```

Comment:
Do not need to push such unused code, check before you pushing

Reply:
Ok, thanks